service/history/workflowExecutionContext.go

```diff
@@ -379,6 +396,14 @@ func (c *workflowExecutionContextImpl) conflictResolveWorkflowExecution(
 		if err != nil {
 			return err
 		}
+		if conflictResolveMode == persistence.ConflictResolveWorkflowModeBypassCurrent {
```

Comment:
this logic should be called right before `ConflictResolveWorkflowExecution`

Reply:
done